.github/workflows/test.yml

```diff
@@ -45,13 +45,44 @@ jobs:
       # Skip Code Search build in CI because it's slow, and we don't use it anyway for testing purposes.
       - run: echo "SKIP_CODE_SEARCH_BUILD=true" >> $GITHUB_ENV
       - run: ./gradlew spotlessCheck
-      - run: ./gradlew check
+      # - run: ./gradlew check
       - name: Upload test report
         if: always()
         uses: actions/upload-artifact@v4
         with:
           name: test-report
           path: build/reports/tests/
-      - run: ./gradlew buildPlugin
-      - run: ./gradlew runPluginVerifier -Pvalidation=lite
+      - run: ./gradlew buildPlugin -x test
+      # - run: ./gradlew runPluginVerifier -Pvalidation=lite
       - run: ./gradlew --stop
+      - uses: actions/upload-artifact@v4
+        with:
+          name: plugin.zip
+          path: './build/distributions/Sourcegraph-*.zip'
+  plugin-verifier:
+    name: IntelliJ Plugin Verifier
+    runs-on: ubuntu-latest
+    needs:
+      - test
+    steps:
+      - name: Free disk space
+        run: |
+          df -h
+          sudo rm -rf /usr/share/dotnet
+          sudo rm -rf /opt/ghc
+          sudo rm -rf "/usr/local/share/boost"
+          df -h
+      - uses: actions/download-artifact@v4
+        with:
+          name: plugin.zip
+      - name: Verify Plugin on IntelliJ Platforms
+        uses: ChrisCarini/intellij-platform-plugin-verifier-action@latest
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          plugin-location: plugin.zip
+          ide-versions: |
+            ideaIC:2021.1
```

Comment:
That is downside, today we can control it from gradle.
What is the problem with running `./gradlew runPluginVerifier -Pvalidation=lite` as a separate job?

Reply:
The `runPluginVerifier` task depends on `buildPlugin`. On a separate runner, this would mean total rebuild, and we wouldn't gain anything.

Reply:
And how that action is supposed to verify the plugin without building it?
I think it build is as well:  https://github.com/ChrisCarini/intellij-platform-plugin-verifier-action?tab=readme-ov-file#plugin-location